src/main/java/de/cismet/cids/custom/objecteditors/wunda_blau/EmobradBetreiberEditor.java

```diff
@@ -0,0 +1,1018 @@
+        // throw new UnsupportedOperationException("Not supported yet."); To change body of generated methods, choose
+        // Tools | Templates.
+    }
+
+    /**
+     * DOCUMENT ME!
+     */
+    private void refreshValidTel() {
+        ftxtTelefon.setValue(telPatternFormatter.getLastValid());
+    }
+
+    /**
+     * DOCUMENT ME!
+     *
+     * @param  okValue  DOCUMENT ME!
+     */
+    private void saveValidTel(final Object okValue) {
+        telPatternFormatter.setLastValid(okValue);
+    }
+
+    /**
+     * DOCUMENT ME!
+     *
+     * @param  field  DOCUMENT ME!
+     * @param  fall   DOCUMENT ME!
+     */
+    private void checkName(final String field, final otherTableCases fall) {
+        // Worker Aufruf, ob das Objekt schon existiert
+        valueFromOtherTable(
+            TABLE_NAME,
+            " where "
+                    + field
+                    + " ilike '"
+                    + txtName.getText().trim()
+                    + "' and "
+                    + FIELD__ID
+                    + " <> "
+                    + cidsBean.getProperty(FIELD__ID),
+            fall);
+    }
+
+    /**
+     * DOCUMENT ME!
+     */
+    private void checkAttributes() {
+        checkName(FIELD__NAME, otherTableCases.redundantAttName);
+        checkName(FIELD__SCHLUESSEL, otherTableCases.redundantAttKey);
+    }
+    /**
+     * DOCUMENT ME!
+     *
+     * @param  url        DOCUMENT ME!
+     * @param  showLabel  DOCUMENT ME!
+     */
+    private void checkUrl(final String url, final JLabel showLabel) {
+        showLabel.setIcon(statusFalsch);
+        showLabel.setCursor(new Cursor(Cursor.DEFAULT_CURSOR));
+        final SwingWorker<Boolean, Void> worker = new SwingWorker<Boolean, Void>() {
+
+                @Override
+                protected Boolean doInBackground() throws Exception {
+                    return WebAccessManager.getInstance().checkIfURLaccessible(new URL(url));
+                }
+
+                @Override
+                protected void done() {
+                    final Boolean check;
+                    try {
+                        check = get();
+                        if (check) {
+                            showLabel.setIcon(statusOk);
+                            showLabel.setCursor(new Cursor(Cursor.HAND_CURSOR));
+                        } else {
+                            showLabel.setIcon(statusFalsch);
+                            showLabel.setCursor(new Cursor(Cursor.DEFAULT_CURSOR));
+                        }
+                    } catch (InterruptedException | ExecutionException e) {
+                        showLabel.setIcon(statusFalsch);
+                        showLabel.setCursor(new Cursor(Cursor.DEFAULT_CURSOR));
+                        LOG.warn("URL Check Problem in Worker.", e);
+                    }
+                }
+            };
+        if (worker_url != null) {
+                worker_url.cancel(true);
+            }
```

Comment:
Auf dem Worker wird `.cancel()` aufgerufen, aber es wird in der `done()-Methode` nirgends `isCancelled()` abgefragt. Ist das so gewollt ?